utils/log.py

```diff
@@ -476,7 +476,7 @@ def art_log(self, level_name, message, kwargs):
             message = str(message)
         art_log_record = {
             'level': level_name,
-            'message': message.encode("ascii", "xmlcharrefreplace"),
+            'message': message.decode('utf-8').encode("ascii", "xmlcharrefreplace"),
```

Comment:
Rewrote all messages to be first decoded (with 'ignore') and then encoded (with xmlcharref).

I still suspect there is a better, out-of-the-box solution for this, lets see if it helps with those crashes